pepper-apis/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -496,6 +527,25 @@ private static void start() {
         LOG.info("ddp startup complete");
     }
 
+    private static void setupRateLimits(int maxRequestsPerSecond, int burst) {
+
+        DoSFilter rateLimitFilter = new DoSFilter();
+        rateLimitFilter.setEnabled(true);
+        rateLimitFilter.setTooManyCode(503);
```

Comment:
Is this the HTTP code client gets when they hit limit? I think we had `429` previously in nginx.

Reply:
Yes--I'll change back.